.gitignore

```diff
@@ -1,2 +1,10 @@
 .idea
-rsrc.syso
\ No newline at end of file
+rsrc.syso
+
+# Linux build and package
+builds/linux/appdir/*

```

Comment:
Nit: could you move these to a `.gitignore` file in `builds/linux` directory similar to `builds/windows`? 
Also the .gitkeep in the `appdir/usr/bin/.gitkeep` directory can be replaced with `.gitignore` with the rules `*`, `!.gitignore`

Reply:
Wow TIL that `.gitkeep` isn't really a git thing. Replaced with `.gitignore` and moved the other one to `build/linux/`